CoreOpenGLDisplay/src/au/gov/asd/tac/constellation/visual/opengl/utilities/SharedDrawable.java

```diff
@@ -222,17 +222,6 @@ public static void updateGlyphTextureController(final GL3 glCurrent) {
         }
     }
 
-    /**
```

Comment:
Was this method removed because it wasn't used?

Reply:
Yeah, it was calling the two methods that SQ flagged as empty. I took a look at it and it was never used within the project

Reply:
This is a static method, perhaps it was intended as a sort of utility function?

Reply:
I think so @cygnus-x-1. I put in a break point here and in the `dispose()` of `GlyphManagerOpenGLController` and wasn't able to see any of them being executed so I concur that they are not used - even magically.

I thought I'd create an Action in Experimental -> Developer -> Clear Glyph Textures Buffer which basically calls:

`SharedDrawable.deleteTextures();`

I then got an exception which said "There is no OpenGL context current in the current thread."

So I modified the method to look like this:

```
        gl.getContext().makeCurrent();
        gl.glDeleteTextures(1, new int[]{iconTextureName}, 0);
        glyphTextureController.dispose(gl);
        gl.getContext().release();
```

Running this seemed fine but when I then tried to create a new graph I got this exception:

```
com.jogamp.opengl.GLException: Thread[AWT-EventQueue-1,6,main] glGetError() returned the following error codes after a call to glBindTexture(<int> 0x8C1A, <int> 0x1): GL_INVALID_OPERATION ( 1282 0x502), 
	at com.jogamp.opengl.DebugGL4bc.writeGLError(DebugGL4bc.java:31781)
	at com.jogamp.opengl.DebugGL4bc.glBindTexture(DebugGL4bc.java:730)
	at au.gov.asd.tac.constellation.visual.opengl.utilities.GLTools.loadSharedIconTextures(GLTools.java:605)
	at au.gov.asd.tac.constellation.visual.opengl.utilities.GLTools.loadSharedIconTextures(GLTools.java:799)
	at au.gov.asd.tac.constellation.visual.opengl.renderer.batcher.IconBatcher.updateIconTexture(IconBatcher.java:117)
	at au.gov.asd.tac.constellation.visual.opengl.renderer.GraphRenderable.lambda$getChangeProcessor$1(GraphRenderable.java:164)
	at au.gov.asd.tac.constellation.visual.opengl.renderer.GraphRenderable.lambda$update$29(GraphRenderable.java:376)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
	at au.gov.asd.tac.constellation.visual.opengl.renderer.GraphRenderable.update(GraphRenderable.java:375)
	at au.gov.asd.tac.constellation.visual.opengl.renderer.GLRenderer.lambda$display$2(GLRenderer.java:165)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
	at au.gov.asd.tac.constellation.visual.opengl.renderer.GLRenderer.display(GLRenderer.java:164)
	at jogamp.opengl.GLDrawableHelper.displayImpl(GLDrawableHelper.java:692)
	at jogamp.opengl.GLDrawableHelper.display(GLDrawableHelper.java:674)
	at com.jogamp.opengl.awt.GLCanvas$11.run(GLCanvas.java:1428)
	at jogamp.opengl.GLDrawableHelper.invokeGLImpl(GLDrawableHelper.java:1293)
Caused: com.jogamp.opengl.GLException: Caught GLException: Thread[AWT-EventQueue-1,6,main] glGetError() returned the following error codes after a call to glBindTexture(<int> 0x8C1A, <int> 0x1): GL_INVALID_OPERATION ( 1282 0x502),  on thread AWT-EventQueue-1
	at com.jogamp.opengl.GLException.newGLException(GLException.java:76)
	at jogamp.opengl.GLDrawableHelper.invokeGLImpl(GLDrawableHelper.java:1327)
	at jogamp.opengl.GLDrawableHelper.invokeGL(GLDrawableHelper.java:1147)
	at com.jogamp.opengl.awt.GLCanvas$12.run(GLCanvas.java:1442)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:303)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:770)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:740)
	at org.netbeans.core.TimableEventQueue.dispatchEvent(TimableEventQueue.java:136)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused: java.lang.RuntimeException
	at com.jogamp.common.util.awt.AWTEDTExecutor.invoke(AWTEDTExecutor.java:58)
	at jogamp.opengl.awt.AWTThreadingPlugin.invokeOnOpenGLThread(AWTThreadingPlugin.java:103)
	at jogamp.opengl.ThreadingImpl.invokeOnOpenGLThread(ThreadingImpl.java:201)
	at com.jogamp.opengl.Threading.invokeOnOpenGLThread(Threading.java:202)
	at com.jogamp.opengl.Threading.invoke(Threading.java:221)
	at com.jogamp.opengl.awt.GLCanvas.display(GLCanvas.java:503)
	at au.gov.asd.tac.constellation.visual.opengl.renderer.GLVisualProcessor.performVisualUpdate(GLVisualProcessor.java:196)
	at au.gov.asd.tac.constellation.utilities.visual.VisualProcessor.lambda$update$0(VisualProcessor.java:211)
[catch] at java.base/java.lang.Thread.run(Thread.java:834)
```

I think given the method isn't used and has serious side effects as it is then its not worth keeping IMO. What do you think @cygnus-x-1 @algol60  @formalhaut69 ?